apps/mobile/src/utils/remoteConfig/atoms.ts

```diff
@@ -0,0 +1,95 @@
+import {atom} from 'jotai'
```

Comment:
Moved into it's own subdirectory to make it more generic. This way we can use remoteConfig values in more convenient ways